modules/inflate_geometry.py

```diff
@@ -0,0 +1,98 @@
+"""Functions for manipulating polygon geometry"""
+
+import math
+import numpy as np
+
+
+def inflate_polygon(vertices: np.ndarray, scale_distance: int) -> np.ndarray:
+    """
+    Given a list of vertices representing a polygon geometry, offset the vertices such that the perpendicular
+    distance between the original and offset edges is equal to the scale distance.
+
+    Parameters
+    -----------
+    vertices: np.ndarray
+        List of vertices
+    scale_distance: Distance (in meters) to offset the vertices by
+
+    Returns
+    -------
+    np.ndarray: List of the offset vertices
+    """
+    # Referance radius for altitude (Radius of earth = 6371 km):
+    r_ref = 6371 * (10**3)
```

Comment:
i think this method may create too much inaccuracy. can we write some unit tests to test the results? also just to verify that it still works well in the event we refactor repo

Reply:
Are you referring to the inflation method as a whole, or just the method of converting between LLA and XYZ?

Reply:
just this method of converting LLA and XYZ, but dont worry about it for now, we should make unit tests and see the results